ProjectLighthouse/Migrations/20211108100949_GameSpecificPlayCountsRedo.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ProjectLighthouse.Migrations
+{
+    public partial class GameSpecificPlayCountsRedo : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
+        }
+    }
+}
```

Comment:
This whole file can be removed since it doesn't do anything.

Reply:
Fixed in https://github.com/LBPUnion/project-lighthouse/pull/38/commits/f93c9c5e329d9f2b8113f1d28015fd979ba0a209